src/services/messengerSync/eventListener.js

```diff
@@ -29,14 +29,34 @@ const handleTeamChanged = async (team) => {
 	}
 };
 
+const handleUserCreated = async (user) => {
+	await channel.assertQueue(QUEUE_INTERNAL, {
+		durable: true,
+	});
+	const message = JSON.stringify({ userId: user._id, fullSync: true });
+	channel.sendToQueue(QUEUE_INTERNAL, Buffer.from(message), { persistent: true });
+};
+
+const handleUserRemoved = async (user) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['user' is defined but never used. Allowed unused args must match /app|req|res|next|options|params/u.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5362179)